src/script/team/TeamService.ts

```diff
@@ -107,6 +107,12 @@ export class TeamService {
         [FEATURE_KEY.SSO]: {
           status: FeatureStatus.ENABLED,
         },
+        [FEATURE_KEY.MLS]: {
+          config: {
+            allowList: [],
+          },
+          status: FeatureStatus.ENABLED,
```

Comment:
Let's have it disabled by default

Reply:
https://github.com/wireapp/wire-webapp/pull/13131/commits/64aaf1b2e7c7e2263a2464962be84e3688d63a7b